src/App.tsx

```diff
@@ -6,9 +6,7 @@ function App() {
     <div className="App">
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
+        <p>Hey</p>
```

Comment:
```suggestion
        <p>Hi</p>
```

![image](https://github.com/TheaAleksovDev/ReactProject/assets/26555796/e735da40-6b95-436b-82f7-cff717c5b0a5)

```js
const num = (a, b) => {
   return a + b
}
```